css/main.css

```diff
@@ -0,0 +1,726 @@
+  font-family: nyt-imperial, georgia, 'times new roman', times, serif;
+}
+
+/* nav */
+
+.times-logo {
+  height: 25px !important;
+  width: 200px !important;
+  padding-top: 0;
+  padding-left: 70px;
+}
+
+.line {
+  width: 100px;
+}
+
+.icons {
+  width: 10px;
+  height: 20px;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.hamb-logo {
+  width: 25px;
+  height: auto;
+}
+
+.icons li {
+  margin-right: 20px;
+}
+
+.nav {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: white;
+  justify-content: space-between;
+  border: 1px solid rgb(204, 204, 204);
+  padding-left: 10px;
+}
+
+.buttons {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+  padding-top: 10px;
+  margin-right: 20px !important;
+}
+
+.nav-bttn {
+  width: 100px;
+  height: 25px;
+  background-color: rgb(86, 123, 149);
+  color: white;
+  border-radius: 3px;
+  font-size: 10px;
+  margin-left: 10px;
+  border: 1px solid rgb(86, 123, 149);
+}
+
+/* main */
+
+main {
+  height: 100vh;
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas: "a1" "a2" "a3" "a4" "a5" "a6" "a7" "a8" "a9" "a10";
+}
+
+#article-heading {
+  text-align: left;
+  font-style: italic;
+  font-size: 1.9375rem;
+  font-weight: 700;
+  line-height: 2.25rem;
+}
+
+.a1 {
+  background-color: white;
+  margin-left: auto;
+  width: 100%;
+  margin-right: auto;
+  grid-area: a1;
+  align-items: center;
+}
+
+.a1 p {
+  margin-left: 400px;
+  margin-right: 400px;
+  line-height: 1.5625rem;
+  text-align: left;
+  font-size: 1.3rem;
+}
+
+.title-img {
+  width: 800px;
+  height: auto;
+}
+
+.top-title {
+  text-align: center;
+}
+
+.top-text {
+  text-align: left;
+  margin-right: 350px;
+  margin-left: 270px;
+  color: #666;
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+}
+
+.top-text-second {
+  text-align: left;
+  margin-right: 330px;
+  margin-left: 270px;
+  color: #666;
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  padding-left: 110px;
+  padding-right: 50px;
+}
+
+.a2 {
+  grid-area: a2;
+  background-color: white;
+  font-size: 1.125rem;
+}
+
+.a2 p {
+  margin-left: 400px;
+  margin-right: 400px;
+  line-height: 1.5625rem;
+  text-align: left;
+  font-size: 1.3rem;
+}
+
+.a3 {
+  grid-area: a3;
+  background-color: white;
+  font-size: 1.125rem;
+}
+
+.a3 p {
+  margin-left: 400px;
+  margin-right: 400px;
+  line-height: 1.5625rem;
+  text-align: left;
+  font-size: 1.3rem;
+}
+
+.a4 {
+  grid-area: a4;
+  background-color: white;
+  font-size: 1.125rem;
+}
+
+.a4 p {
+  margin-left: 400px;
+  margin-right: 400px;
+  line-height: 1.5625rem;
+  text-align: left;
+  font-size: 1.3rem;
+}
+
+.a5 {
+  grid-area: a5;
+  background-color: white;
+  font-size: 1.125rem;
+}
+
+.a5 p {
+  margin-left: 400px;
+  margin-right: 400px;
+  line-height: 1.5625rem;
+  text-align: left;
+  font-size: 1.3rem;
+}
```

Comment:
- [ ] **MANDATORY** You also repeat the same stylings for many clases, for good coding practices and to follow the DRY (Don't repeat yourself) rule, you have 2 options this will also help you when you have to do a change for all similar elements:
- use multiselection:
```
.a1 p, .a1 p, .a2 p, .a4 p {
   ... (Your styles)
}
```
* OR  you can use multiple classes for any element, so for example,

  If you use this line in many classes:
  `color: #fff;` and you use it for .class1, .class2, and .class3 
  you can put it inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px and color of #fff -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px and color of #fff -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px and color of #fff -->
  ...
  ```